data/python/use-cases/reportstats/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM python:3.13
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -yqq && \
+    apt-get dist-upgrade -yqq && \
+    apt-get install -yqq git
+    libbz2-dev \
```

Hadolint (2.14.0)

[error] unexpected 'l'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)